common/views/components/VenueHours/VenueHours.js

```diff
@@ -97,153 +117,112 @@ const VenueHours = ({ venue, isInList }: Props) => {
   };
 
   return (
-    <div className="row">
-      <div className="container">
-        <div className="grid">
-          {isInList && (
-            <>
-              <div
-                className={classNames({
-                  [spacing({ s: 4 }, { padding: ['bottom'] })]: true,
-                  [grid({
-                    s: 12,
-                    m: 12,
-                    l: 11,
-                    xl: 10,
-                    shiftL: 1,
-                    shiftXl: 2,
-                  })]: true,
-                  'is-hidden-s': true,
-                })}
-              >
-                <Divider extraClasses="divider--keyline divider--pumice" />
-              </div>
-              <VenueHoursImage>
-                <UiImage
-                  contentUrl={
-                    venueAdditionalInfo[venue.name.toLowerCase()].image
-                  }
-                  width={1600}
-                  height={900}
-                  alt=""
-                  tasl={null}
-                  sizesQueries={null}
-                  extraClasses=""
-                  showTasl={false}
-                />
-              </VenueHoursImage>
-            </>
-          )}
-          <div
+    <>
+      {isInList && (
+        <>
+          <Divider
+            extraClasses={classNames({
+              'divider--keyline': true,
+              'divider--pumice': true,
+              'is-hidden-s': true,
+              [spacing({ s: 4 }, { margin: ['bottom'] })]: true,
+            })}
+          />
+          <VenueHoursImage>
+            <UiImage
+              contentUrl={venueAdditionalInfo[venue.name.toLowerCase()].image}
+              width={1600}
```

Comment:
Shouldn't this be responsive?

Reply:
It is but was missing the sizes attribute, have put that in now